app/scripts/controllers/visualizerModal.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+
+angular.module('openhimConsoleApp')
+  .controller('VisualizerModalCtrl', function ($http, $scope, $modalInstance, $timeout, Api, Notify, Alerting, settingsStore, visualizers) {
+
+
+    /***************************************************/
+    /**         Initial page load functions           **/
+    /***************************************************/
+
+    $scope.ngError = {};
+    $scope.validationRequiredMsg = 'This field is required.';
+
+    // get/set the users scope whether new or update
+    if (settingsStore) {
+      $scope.update = true;
+      
+      
+    }else{
+      $scope.update = false;
+
+      // create visualizer settings properties
+      $scope.settings = {};
+      $scope.settings.visualizer = {};
```

Comment:
Oh wait never mind I see what you mean - we just remove the 'visualizer' part but still keep the viewModel.